MaxText/train.py

```diff
@@ -356,9 +361,13 @@ def main(argv: Sequence[str]) -> None:
       stack_trace_to_cloud = config.stack_trace_to_cloud,
       stack_trace_interval_seconds = config.stack_trace_interval_seconds))
   diagnostic_config = diagnostic_configuration.DiagnosticConfig(debug_config)
+  job_start_time = datetime.datetime.now()
   with diagnostic.diagnose(diagnostic_config):
     train_loop(config)
-
+  job_end_time = datetime.datetime.now()
+  goodput.goodput_log_job_runtime(config, job_end_time - job_start_time)
+  total_goodput = goodput.get_job_goodput(config)
+  max_logging.log(f"Total job goodput: {total_goodput:.2f}%")
```

Comment:
Are we sure we want to print this? Because this run doesn't know the overall goodput -- the goodput depends on fusing multiple runs together.